packages/components/buttons/accessible-button/src/accessible-button.tsx

```diff
@@ -75,7 +75,7 @@ type TAccessibleButtonProps = {
   /**
    * Any HTML attributes to be forwarded to the HTML element.
    */
-  buttonAttributes?: Record<string, string>;
```

Comment:
this needs to support `unknown`. since buttonAttributes can be `{ [string]: boolean }`